Tests/test_file_png.py

```diff
@@ -778,6 +778,15 @@ class MyStdOut:
         with Image.open(mystdout) as reloaded:
             assert_image_equal_tofile(reloaded, TEST_PNG_FILE)
 
+    def test_end_truncated_filed(self):
+        ImageFile.LOAD_TRUNCATED_IMAGES = True
+        try:
+            img = Image.open("Tests/images/end_trunc_file.png")
+            img.load()
+            assert img is not None
```

Comment:
```suggestion
            with Image.open("Tests/images/end_trunc_file.png") as im:
                assert_image_equal_tofile(im, "Tests/images/hopper.png")
```
Two suggestions here.
1) This [uses a context manager.](https://pillow.readthedocs.io/en/stable/releasenotes/7.0.0.html#image-del)
2) This tests the image contents are correct.

Reply:
ok, i will resolve it.